src/calendar/tests/unit/Calendar.ts

```diff
@@ -161,6 +161,130 @@ const expected = function(popupOpen = false, selectedIndex = -1, weekdayLabel =
 	]);
 };
 
+const expectedMondayStart = function(popupOpen = false, selectedIndex = -1, weekdayLabel = '', customMonthLabel = false, describedby = '') {
+	const overrides = describedby ? { 'aria-describedby': describedby } : {};
+	dateIndex = -1;
+	let weekdayNames = [];
+	for (let i = 0; i < DEFAULT_WEEKDAYS.length; i++) {
+		weekdayNames.push(DEFAULT_WEEKDAYS[(i + 1) % 7]);
+	}
+	return v('div', {
+		classes: css.root,
+		dir: '',
+		lang: null,
```

Comment:
The latest changes to the I18n mixin in `widget-core` removed unset `dir` and `lang` attributes. If you rebase and remove those two lines, tests should pass